plugin/src/main/java/jenkins/plugins/openstack/compute/JCloudsSlaveTemplate.java

```diff
@@ -87,10 +89,11 @@
     private transient @Deprecated @SuppressWarnings("DeprecatedIsStillUsed") String availabilityZone;
 
     @DataBoundConstructor
-    public JCloudsSlaveTemplate(final @Nonnull String name, final @Nonnull String labels, final @CheckForNull SlaveOptions slaveOptions) {
+    public JCloudsSlaveTemplate(final @Nonnull String name, final @Nonnull String labels, Node.Mode mode, final @CheckForNull SlaveOptions slaveOptions) {
         this.name = Util.fixNull(name).trim();
         this.labelString = Util.fixNull(labels).trim();
-
+        this.mode = mode != null ? mode : Node.Mode.NORMAL;
+        LOGGER.info("Constructing new JCloudsSlaveTemplate - mode=" + this.mode);
```

Comment:
This probably does not varant `INFO` level.

Reply:
Sure.  I've changed it to LOGGER.fine now